### DISCUSSION TITLE
RÜTBE KAZANMA YOLLARI

### DISCUSSION THREAD

**Discussion by canrasim** [IBAX-io/go-ibax#719] [Soru Cevap - Türkçe]
RÜTBE KAZANMA YOLLARI HAKKINDA DETAYLI BİLGİ SAHİBİ OLANLAR BU TAG ALTINDA BİRLEŞEBİLİR

**Comment by Serdarkck** [comment]
Rütbe nasıl kazanılır

**Comment by hes91** [comment]
Rütbe kazanmak istiyorsan referansın sağlam olmalı.Yoksa diğer görevler hep stabil.O yüzden referans önemli

**Comment by canrasim** [comment]
öncelik referans ama diğer konularada dikkat çeklemk lazım github gibi

**Comment by canrasim** [comment]
peki referans nasıl kasılabilir

**Reply by Yusufknk** [reply]
Rütbenizi yükseltmek için daha çok puan ve davetli getirmeniz gerekmektedir. Puanları görevlerden çekilişlerden ve  sandıktan kazanabilirsiniz.
